src/components/Chart/Gantt/GanttChart.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect } from 'react'
+import * as am4core from '@amcharts/amcharts4/core'
+import * as am4charts from '@amcharts/amcharts4/charts'
+import { Am4ThemesColorTheme } from '../../utils/Colors'
+function SunburstChart ({ data, endDateForXAxis }) {
+  const chartId = 'chart'
+
+  useEffect(() => {
+    am4core.useTheme(Am4ThemesColorTheme)
+    const colorSet = new am4core.ColorSet()
+    // Add theme colors
+    data = data.map(function (item, index) {
+      // There are 10 colors in the theme, so 
+      // if the index exceeds 9, the last int in
+      // the number is used
+      const number = index > 9 ? index.toString().substr(-1) : index
```

Comment:
Using `index%10` would make the code cleaner and make sure that `number` is actually an integer (currently it will be a string if `index > 9`).

Reply:
skal ske skal ske